src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md

```diff
@@ -2,36 +2,36 @@
 
-The opening of the river trade coincided with another important factor: the growing interest in English beer from the Baltic states, namely Poland and the Russian Empire. It is said that Peter the Great of Russia had brought from England that he visited at the end of the 17th century the love for English beer. No reliable source on the matter survived, and Peter in his own vigorous way had, first of all, established the beer industry in Russia itself[ref:durov-peter-beer](). However, the maritime trade was growing, accelerating further towards the end of the century. Imports to Russia, the majority of which were controlled by Burton, comprised 100 thousand liters in 1750 and more than 1.5 million liters in 1775[ref:{"source":{"alias":"brewing-in-burton","short":"Webster, I. (2018)","extra":["Brewing in Burton upon Trent","ISBN 9781445670553"]}}:9-10]().
+The expansion of river trade coincided with another important factor: the growing interest in English beer from the Baltic states, namely Poland and the Russian Empire.  Peter the Great of Russia had visited England and it is said that he brought with him a love for English beer. No reliable source on the matter has survived, and Peter in his own inimitable way had in fact established the beer industry in Russia itself[ref:durov-peter-beer](). However, maritime trade was growing, accelerating further towards the end of the century. Imports to Russia, the majority of which were controlled by Burton, comprised 100 thousand liters in 1750 and more than 1.5 million liters by 1775[ref:{"source":{"alias":"brewing-in-burton","short":"Webster, I. (2018)","extra":["Brewing in Burton upon Trent","ISBN 9781445670553"]}}:9-10]().
 
-Burtoners owed this expansion to water: first, the River Trent that gave access to British ports; second, the unique qualities of local water sources. Burton's water was hard and rich in calcium and magnesium sulfates. It turned out that this water suited brewing exceptionally well, stimulating the growth of yeasts and allowing for intensive hopping. Burton beer was more carbonated and much more clear than the London one (and it also got the unique ‘Burton snatch’ — the fleeting ‘aroma’ of sulfur that occurred shortly after pouring).
+Burtoners owed this expansion to water. First, the River Trent gave access to British ports. Second, it held utterly unique local water qualities. Burton's water was hard and rich in calcium and magnesium sulfates. It turned out that this water suited brewing exceptionally well, stimulating the growth of yeasts and allowing for intensive hopping. Burton beer was more carbonated and much clearer than London beer (and it also contained the unique ‘Burton snatch’ — the fleeting ‘aroma’ of sulfur that occurred shortly after pouring).
```

Comment:
I read it as 'water from Trent river had unique qualities', which is not true: Burtoners were using boreholes AFAIK.

Reply:
yes!